src/Microsoft.AspNetCore.Razor.Language/IRazorCodeDocumentFeature.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal interface IRazorCodeDocumentFeature : IRazorProjectEngineFeature
```

Comment:
Nothing depends on this being public today. Figured it might as well be internal